frontend/app-development/hooks/queries/useGetAvailableOptionListsFromOrgQuery.test.ts

```diff
@@ -0,0 +1,22 @@
+import { waitFor } from '@testing-library/react';
+import { useGetAvailableOptionListsFromOrgQuery } from './useGetAvailableOptionListsFromOrgQuery';
+import { org } from '@studio/testing/testids';
+import { renderHookWithProviders } from 'app-shared/mocks/renderHookWithProviders';
+import { listOfAvailableOptionListTitlesToImport } from 'app-shared/mocks/optionListDataListMock';
+
+describe('useGetAvailableOptionListsFromOrgQuery', () => {
+  it('Calls getAvailableOptionListDataListsInOrg with correct arguments and returns the data', async () => {
+    const getAvailableOptionListDataListsInOrg = jest
+      .fn()
+      .mockImplementation(() => Promise.resolve(listOfAvailableOptionListTitlesToImport));
+
+    const result = renderHookWithProviders(() => useGetAvailableOptionListsFromOrgQuery(org), {
+      queries: { getAvailableOptionListDataListsInOrg },
+    }).result;
+
+    await waitFor(() => result.current.isPending);
+    expect(getAvailableOptionListDataListsInOrg).toHaveBeenCalledWith(org);
+    await waitFor(() => result.current.isSuccess);
+    expect(result.current.data).toEqual(listOfAvailableOptionListTitlesToImport);
```

Comment:
Is it possible to remove a line here?
```suggestion
    await waitFor(() => result.current.isSuccess);
    expect(getAvailableOptionListDataListsInOrg).toHaveBeenCalledWith(org);
    expect(result.current.data).toEqual(listOfAvailableOptionListTitlesToImport);
```